test/index.js

```diff
@@ -23,12 +23,18 @@ const mockHook = (delay = 0) => ({
     this.taps.map(t => setTimeout(() => t.cb(t.name), delay));
   }
 });
+const mockLogger = {
+  info: fake(),
+  error: fake()
+};
+const createLogger = () => mockLogger;
 const mockCompiler = () => ({
   options: { watch: false },
   hooks: {
     beforeRun: mockHook(),
     afterCompile: mockHook()
-  }
+  },
+  getInfrastructureLogger: createLogger
```

Comment:
The webpack method of creating a logger is to get one off of the current compiler instance.

https://v4.webpack.js.org/api/plugins/#logging